DEPLOY.md

```diff
@@ -10,7 +10,7 @@ This document will explain you how to run a local copy of Pan Docs.
   # Install mdbook using cargo
   cargo install mdbook
   # Remember to add cargo bin directory to your path
-  # Install python3 (on Debian systems)
+  # Install Python 3.9 or earlier; for example, on APT-based systems (Ubuntu, Debian...):
```

Comment:
```suggestion
  # Install Python 3.9 or earlier; for example, on debian-based systems:
```

A system is not apt-based. A package manager can be apt-based.

Reply:
I accepted a [suggestion](https://github.com/gbdev/pandocs/pull/453#discussion_r1020931875) from @ISSOtm so I believe we need his comment here too

Reply:
I would say a system can be called "APT-based" if `apt` is configured for a `/usr` prefix instead of a `/usr/local` or other prefix. While it's theoretically possible to run multiple concurrent package managers, *usually* only one is the "system" package manager, and systems for which APT is that, I would call "APT-based".

And I believe that people running several concurrent package managers will be savvy enough to forgive the inaccuracy :stuck_out_tongue: